functions/payment.js

```diff
@@ -1,282 +0,0 @@
-const admin = require("firebase-admin");
-const functions = require("firebase-functions");
-
-const PROJECT_ID = functions.config().project.id;
-const STRIPE_CONFIG = functions.config().stripe;
```

Comment:
From a quick skim, doesn't look like we've updated the values in `functions/.runtimeconfig.json.example` to account for this change?

Reply:
fixed in ebe967cd50437f138f6685ec3ce64daef7abeb40